src/main/webapp/WEB-INF/tags/admin/search/studentRow.tag

```diff
@@ -111,8 +120,11 @@
       <c:if test="${not empty student.publishedFeedbackSessions}">
         <c:forEach items="${student.publishedFeedbackSessions}" var="session">
           <li class="list-group-item list-group-item-success">
-            <strong>${session.fsName}</strong>
-            <input value="${session.link}" readonly class="form-control">
+            <search:emailFormFields linkTitle="${session.fsName}"
+                                    relatedLink="${session.link}"
+                                    subjectType="Feedback session results published"
+                                    sessionStatus="Published"
+                                    student="${student}"/>
```

Comment:
We've shifted from using custom indentation in our code; as such, it might be better to do the following:

```html
<search:emailFormFields
    linkTitle="${session.fsName}"
    relatedLink="${session.link}"
    subjectType="Feedback session results published"
    sessionStatus="Published"
    student="${student}"/>
```